src/App.tsx

```diff
@@ -239,8 +234,9 @@ export default function App() {
         {/*gives CardEditor the ability to change Deck view */}
         <CardEditor
           dispatch={dispatch}
-          createDSLValue={createDSLValue}
-        ></CardEditor>{" "}
+          program={program}
+          initialText={initialText}
+        ></CardEditor>
```

Comment:
initial ` PROGRAM` (loaded from localStorage) is passed to CardEditor, so that, from there, it always compares the updated `PROGRAM` to the previous, to make CRUD changes